pysonde/thermodynamics.py

```diff
@@ -46,93 +48,141 @@ def convert_rh_to_dewpoint(temperature, rh):
 
 def calc_saturation_pressure(temperature, method="hardy1998"):
     """
-    Calculate saturation water vapor pressure
-
-    Input
-    -----
-    temperature : array
-        array of temperature in Kevlin or dew point temperature for actual vapor pressure
-    method : str
-        Formula used for calculating the saturation pressure
-            'hardy1998' : ITS-90 Formulations for Vapor Pressure, Frostpoint Temperature,
-                Dewpoint Temperature, and Enhancement Factors in the Range –100 to +100 C,
-                Bob Hardy, Proceedings of the Third International Symposium on Humidity and Moisture,
-                1998 (same as used in Aspen software after May 2018)
-
-    Return
-    ------
-    e_sw : array
-        saturation pressure (Pa)
-
-    Examples
-    --------
-    >>> calc_saturation_pressure([273.15])
-    array([ 611.2129107])
-
-    >>> calc_saturation_pressure([273.15, 293.15, 253.15])
-    array([  611.2129107 ,  2339.26239586,   125.58350529])
```

Comment:
Let's keep these. These are not just examples but also serve as doctests. It would actually be good if you could also add one for `wagner_pruss`:

```suggestion
    Examples
    --------
    >>> calc_saturation_pressure([273.15])
    array([ 611.2129107])
    >>> calc_saturation_pressure([273.15, 293.15, 253.15])
    array([  611.2129107 ,  2339.26239586,   125.58350529])
    >>> calc_saturation_pressure([273.15, 293.15, 253.15], method='wagner_pruss')
    array([what, ever, the, values, are])
```